website/content/docs/k8s/k8s-cli.mdx

```diff
@@ -2,73 +2,76 @@
+| <nobr>`-set-file`</nobr>      | String value that specifies the name of an arbitrary config file. This flag is comparable to the `helm install --set-file` <br/> flag. The contents of the file will be used to set a customizable value. You can use the `-set-file` flag multiple times to specify multiple files. <br/> Consul Helm chart values are supported.                                                                | none                                    |
+| <nobr>`-set-string`</nobr>    | String value that enables you to set a customizable string value. This flag is comparable to the `helm install --set-string` <br/> flag. You can use the `-set-string` flag multiple times to specify multiple strings. <br/> Consul Helm chart values are supported.                                                                                                                             | none                                    |
+| <nobr>`-timeout`</nobr>       | Specifies how long to wait for the installation process to complete before timing out. The value is specified with an integer and string value indicating a unit of time. <br/> The following units are supported: <br/> `ms` (milliseconds)<br/>`s` (seconds)<br/>`m` (minutes) <br/>In the following example, installation will timeout after one minute:<br/> `consul-k8s install -timeout 1m` | `10m`                                   |
+| <nobr>`-wait`</nobr>          | Boolean value that determines if Consul should wait for resources in the installation to be ready before exiting the command.                                                                                                                                                                                                                                                                     | `true`                                  |
+| <nobr>`-verbose`, `-v`</nobr> | Boolean value that specifies whether to output verbose logs from the install command with the status of resources being installed.                                                                                                                                                                                                                                                                | `false`                                 |
+| <nobr>`-help`, `-h`</nobr>    | Prints usage information for this option.                                                                                                                                                                                                                                                                                                                                                         | none                                    |
```

Comment:
Here and in other tables, multiple `&nbsp;` were used to keep the markdown table from breaking the first column. I replaced this with `<nobr>` which is better for screen readers and takes up less space.